<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>535</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PP_Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ledt_accid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledt_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_login">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>352</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 11pt &quot;楷体&quot;;
background-color: rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>登   录</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_register">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>302</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>立即注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_register_tip">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>没有账号？</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>535</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/title.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/login_window/title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_portrait">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/portrait.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:img/login_window/portrait.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_quit">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>0</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/quit.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_login_tip">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>331</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 25 9pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
